Silicon/NXP/iMX6Pkg/iMX6CommonDsc.inc

```diff
@@ -850,6 +856,15 @@
   giMXPlatformTokenSpaceGuid.PcdGlobalDataBaseAddress|$(GLOBAL_DATA_BASE_ADDRESS)
   giMXPlatformTokenSpaceGuid.PcdGlobalDataSize|0x1000
 
+########################
+#
+# giMX6TokenSpaceGuid PCDs
+#
+########################
+[PcdsFixedAtBuild.common]
+
+  giMX6TokenSpaceGuid.PcdLCDIFBase|0x021C8000

```

Comment:
This should be wrapped in an !if $(IMX_FAMILY) == IMX6ULL like the other SoC specific addresses.